template/package.json

```diff
@@ -10,90 +10,55 @@
         "build": "preact build",
         "serve": "preact build && preact serve",
         "dev": "preact watch",
-        "lint": "tslint --project src",
+        "lint": "eslint src/**/*.{js,jsx,ts,tsx}",
         "test": "jest ./tests",
         "precommit": "lint-staged"
     },
     "lint-staged": {
-        "*.{js,jsx,css,md,scss}": [
+        "*.{css,md,scss}": [
             "prettier --write",
             "git add"
         ],
-        "*.{ts,tsx}": [
-            "tslint --fix",
+        "*.{js,jsx,ts,tsx}": [
+            "eslint --fix",
             "git add"
         ]
     },
-    "eslintConfig": {
-        "extends": "eslint-config-aerian"
-    },
     "eslintIgnore": [
         "build/*"
     ],
-    "devDependencies": {
-        "@types/webpack-env": "^1.13.9",
-        "@types/jest": "^23.3.10",
-        "husky": "^1.2.0",
-        "identity-obj-proxy": "^3.0.0",
-        "jest": "^23.6.0",
-        "lint-staged": "^8.1.0",
-        "per-env": "^1.0.2",
-        "preact-cli": "^3.0.0-next.19",
-        "preact-render-spy": "^1.3.0",
-        "prettier": "^1.15.3",
-        "ts-jest": "^23.10.5",
-        "ts-loader": "^3.5.0",
-        "tslint": "^5.11.0",
-        "tslint-config-prettier": "^1.17.0",
-        "tslint-consistent-codestyle": "^1.14.1",
-        "tslint-eslint-rules": "^5.4.0",
-        "tslint-react": "^3.6.0",
-        "typescript": "^3.2.1",
-        "typings-for-css-modules-loader": "^1.7.0",
-        "css-loader": "^1.0.1"
+    "peerDependencies": {
+        "prettier": ">1.0.0",
+        "eslint": ">6.0.0"
```

Comment:
do these have any value? I think we could remove them, not sure why we had them before?

Reply:
I don't think they do. Couldn't see how they'd be useful in a template. I'll have them removed.